### PR TITLE
Cleanup unwanted configuration

### PR DESCRIPTION
**Pull request by baijum**
Remove image build for OLM based tests. As of now, there is no OLM based tests.

### CODE REVIEW

**Comment by baijum** [comment]
@Avni-Sharma I have removed the operator registry image also. Please check again.

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml [hunk 1]
```diff
@@ -17,22 +13,6 @@ resources:
       memory: 250Mi
       cpu: 100m
 
-images:
```
Comment (isutton) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml]: Why are we removing this key?

Comment (baijum) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml]: As of now, we don't use these images for any tests.

**Pull request merged**